workshops/models.py

```diff
@@ -159,6 +159,35 @@ def self_organized_workshops_per_year_remaining(self):
 #------------------------------------------------------------
 
 
+class Sponsorship(models.Model):
+    '''Represent sponsorship from a host for an event.'''
+
+    SPONSORSHIP_SLABS = (
```

Comment:
@gvwilson @pbanaszkiewicz I would like to know whether a choice field would be better suited for SWC/DC workshops or a plain simple `IntegerField`?
If we go with choices, what are the slabs in which SWC/DC accept sponsorship.